core/src/main/java/org/microshed/testing/jaxrs/RestClientBuilder.java

```diff
@@ -18,26 +18,19 @@
  */
 package org.microshed.testing.jaxrs;
 
-import java.nio.charset.StandardCharsets;
-import java.util.Arrays;
-import java.util.Base64;
-import java.util.Collections;
-import java.util.HashMap;
-import java.util.List;
-import java.util.Map;
-import java.util.Objects;
-
 import jakarta.ws.rs.ApplicationPath;
 import jakarta.ws.rs.core.Application;
 import jakarta.ws.rs.ext.MessageBodyReader;
 import jakarta.ws.rs.ext.MessageBodyWriter;
-
 import org.apache.cxf.jaxrs.client.JAXRSClientFactoryBean;
 import org.junit.platform.commons.support.AnnotationSupport;
 import org.junit.platform.commons.support.ReflectionSupport;
 import org.microshed.testing.ApplicationEnvironment;
 import org.microshed.testing.internal.InternalLogger;
 
+import java.nio.charset.StandardCharsets;
+import java.util.*;
```

Comment:
Please revert back to importing by class. Ambiguous imports is bad practice we should avoid.

Reply:
This is still not resolved.  Cannot merge with ambiguous imports it leads to a cluttered namespace.

Reply:
It is though?